common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -429,6 +428,11 @@ div.problem {
       }
     }
 
+    // Fix for formulaequationinput
```

Comment:
And here I was thinking I was being good for putting a comment in the sass at all! ;-)